manager/src/main/java/com/redhat/service/smartevents/manager/dao/ProcessorDAO.java

```diff
@@ -182,4 +182,10 @@ public Long countByBridgeIdAndCustomerId(String bridgeId, String customerId) {
         return countProcessorsOnBridge(p);
     }
 
+    public Long countUserVisibleByBridgeId(String bridgeId) {
+        TypedQuery<Long> namedQuery = getEntityManager().createNamedQuery("PROCESSOR.countUserDefineProcessorByBridgeId", Long.class);
+        namedQuery.setParameter(Processor.BRIDGE_ID_PARAM, bridgeId);
+        namedQuery.setParameter("processorType", USER_VISIBLE_PROCESSOR_TYPES);
```

Comment:
Can we make these constants like the `Process.BRIDGE_ID_PARAM` for consistency.
Thanks!

Reply:
done